app.go2tv.go2tv.yml

```diff
@@ -0,0 +1,39 @@
+app-id: app.go2tv.go2tv
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: *runtime-version
```

Comment:
this won't work, follow the docs https://docs.flatpak.org/en/latest/extension.html#loading-existing-extensions

Reply:
ty, updated